src/Microsoft.ML.LightGBM/LightGbmArguments.cs

```diff
@@ -371,6 +371,14 @@ public enum EvalMetricType
         [TlcModule.SweepableDiscreteParam("CatL2", new object[] { 0.1, 0.5, 1, 5, 10 })]
         public double CatL2 = 10;
 
+        [Argument(ArgumentType.Multiple,
+            HelpText = "Sets the constraints for monotonic features. This is a 0 based index for each feature in " +
+            "the features column. A keyword of 'pos' for positive constraint or 'neg' for negative constraint is " +
+            "specified followed by a range. For example, pos:0-2 neg:3,5 will apply a positive constraint to the " +
+            "first three features and a negative constraint to the 4th and 6th feature. If feature index is not specified, " +
+            "then no constraint will be applied.")]
+        public string[] MonotoneConstraints;
```

Comment:
>public string[] MonotoneConstraints; [](start = 8, length = 36)

A) please unblock and run test RegenerateEntryPointCatalog on your local machine and push changes in core_ep-list.tsv.
B) How it would handle null value? I don't see test covering that.
C) How it would handle gibberish values? Like `Bubba, TabbyCat` ? What kind of exception it throw, was that exception be enough to understand something is wrong with this parameter?